dev/integration_tests/android_semantics_testing/test_driver/main_test.dart

```diff
@@ -198,8 +198,16 @@ void main() {
       });
 
       test('Checkbox has correct Android semantics', () async {
+        Future<AndroidSemanticsNode> getCheckboxSemantics(String key) async {
+          return getSemantics(
+            find.descendant(
+              of: find.byValueKey(key),
+              matching: find.byType('_CheckboxRenderObjectWidget'),
```

Comment:
Why does this need to be against `_CheckboxRenderObjectWidget` and not just the key?

Reply:
Because the key corresponds to a `RenderMouseRegion` now that I've added the `MouseRegion`, so we have to dig down to find the render object for the checkbox which adds the semantics.